sympy/integrals/tests/test_heurisch.py

```diff
@@ -244,12 +245,15 @@ def test_pmint_logexp():
 
     assert ratsimp(heurisch(f, x)) == g
 
-@XFAIL  # there's a hash dependent failure lurking here
+
+#@XFAIL  # there's a hash dependent failure lurking here
+# Seems to work on Python 3, but keeping the comment above just in case
 def test_pmint_erf():
-    f = exp(-x**2)*erf(x)/(erf(x)**3 - erf(x)**2 - erf(x) + 1)
-    g = sqrt(pi)*log(erf(x) - 1)/8 - sqrt(pi)*log(erf(x) + 1)/8 - sqrt(pi)/(4*erf(x) - 4)
+    if PY3:
+        f = exp(-x**2)*erf(x)/(erf(x)**3 - erf(x)**2 - erf(x) + 1)
+        g = sqrt(pi)*log(erf(x) - 1)/8 - sqrt(pi)*log(erf(x) + 1)/8 - sqrt(pi)/(4*erf(x) - 4)
 
-    assert ratsimp(heurisch(f, x)) == g
+        assert ratsimp(heurisch(f, x)) == g
```

Comment:
I don't know if it can be done with pytest compatibility, but with the non-pytest runner you can `raise XFail` to keep it XFAIL in Python 2.

Otherwise you can add

```py
if not PY3:
    test_pmint_erf = XFAIL(test_pmint_erf)
```

below the function definition.

Reply:
OK!